### PR TITLE
Updating qe-tool

### PR DESCRIPTION
**Pull request by dshchedr**

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1243,7 +1243,7 @@ k8s.io/utils v0.0.0-20190801114015-581e00157fb1/go.mod h1:sZAwmy6armz5eXlNoLmJcl
 k8s.io/utils v0.0.0-20190920012459-5008bf6f8cd6/go.mod h1:sZAwmy6armz5eXlNoLmJcl4F1QuKu7sr+mFQ0byX7Ew=
 k8s.io/utils v0.0.0-20191114184206-e782cd3c129f h1:GiPwtSzdP43eI1hpPCbROQCCIgCuiMMNF8YUVLF3vJo=
 k8s.io/utils v0.0.0-20191114184206-e782cd3c129f/go.mod h1:sZAwmy6armz5eXlNoLmJcl4F1QuKu7sr+mFQ0byX7Ew=
-kubevirt.io/qe-tools v0.1.4 h1:q4xvpLIyO6Ljl7u4Bu27EM2guNAuPolOqwCYJ2QeIik=
+kubevirt.io/qe-tools v0.1.4 h1:t1O6OtLV1auU3J60XXDnzX2JxNQ5kWJTQAHaBUGUbTY=
```
Comment (slintes) [inline go.sum:1246]: see comment here: https://github.com/openshift-kni/cnf-features-deploy/pull/139/files#r404264706

File: vendor/kubevirt.io/qe-tools/pkg/ginkgo-reporters/polarion_reporter.go [hunk 2]
```diff
@@ -121,6 +125,14 @@ func (reporter *PolarionReporter) SpecSuiteWillBegin(config config.GinkgoConfigT
 		reporter.Suite.Properties.Property, "polarion-custom-isautomated", "True")
 	reporter.Suite.Properties.Property = addProperty(
 		reporter.Suite.Properties.Property, "polarion-testrun-status-id", "inprogress")
+	if reporter.TestRunTemplate != "" {
+		reporter.Suite.Properties.Property = addProperty(
+			reporter.Suite.Properties.Property, "polarion-testrun-template-id", reporter.TestRunTemplate)
+	}
+	if reporter.TestRunTemplate != "" {
```
Comment (slintes) [inline vendor/kubevirt.io/qe-tools/pkg/ginkgo-reporters/polarion_reporter.go:132]: I guess this should be `TestRunTitle`?

Comment (dshchedr) [reply vendor/kubevirt.io/qe-tools/pkg/ginkgo-reporters/polarion_reporter.go:132]: ah, thanks. very good catch!

**Pull request merged**